docs/user-guide/rug/generators.md

```diff
@@ -16,46 +16,135 @@ Let's look more closely at what makes a project a Rug generator.
 
 ## Anatomy of a Generator
 
-Suppose we have a model project for creating new documentation
-projects based on the [MkDocs][mkdocs] static site generator.  The
-contents of the model project are the following.
+Suppose we have a model project our team clones to quickly get the skeleton of a 
+[Spring Bot Rest Service][springrest].  The contents of the model project are 
+the following.
 
-[mkdocs]: http://www.mkdocs.org/
+[springrest]: https://spring.io/guides/gs/rest-service/
 
 ```console
-mkdocs-project
+spring-boot-rest-basic
+    ├── .gitignore
+    ├── pom.xml
     ├── README.md
-    ├── docs
-    │   └── index.md
-    ├── mkdocs.yml
-    └── requirements.txt
+    ├── src
+    │   ├── main
+    │   │   ├── java
+    │   │   │   └── com
+    │   │   │       └── company
+    │   │   │           ├── HomeController.java
+    │   │   │           ├── MyRestServiceApplication.java
+    │   │   │           └── MyRestServiceConfiguration.java
+    │   │   └── resources
+    │   │       ├── application.properties
+    │   │       └── logback.xml
+    │   └── test
+    │       └── java
+    │           └── com
+    │               └── company
+    │                   ├── MyRestServiceApplicationTests.java
+    │                   ├── MyRestServiceOutOfContainerIntegrationTests.java
+    │                   └── MyRestServiceWebIntegrationTests.java
+    └── .travis.yml
 ```
 
-To enable this project as a Rug generator, we simply add a `.atomist`
-directory to the root of the project.
+To turn our project into a [Rug project][rugproj], we can use the 
+`ConvertExistingProjectToGenerator` Rug generator in [rug-editors][convert] to 
+add all the necessary directories and files:
+
+[rugproj]: projects.md
+[convert]: https://github.com/atomist-rugs/rug-editors#convertexistingprojecttogenerator
 
 ```console
-mkdocs-project
-    ├── .atomist
-    │   ├── manifest.yml
-    │   ├── package.json
-    │   ├── editors
-    │   │   └── NewMkDocsDocumentationProject.ts
-    │   └── tests
-    │       └── NewMkDocsDocumentationProject.rt
-    ├── README.md
-    ├── docs
-    │   └── index.md
-    ├── mkdocs.yml
-    └── requirements.txt
+$ rug edit atomist-rugs:rug-editors:ConvertExistingProjectToGenerator \
+    archive_name=spring-boot-service \
+    group_id=com.company.rugs \
+    version=0.13.0 \
```

Comment:
No, the version parameter to the editor specifies the initial version of the project being created, not the version of rug to use.